### PR TITLE
8228 reporting market loading err

### PR DESCRIPTION
**Pull request by bthaile**
[Clubhouse Story](https://app.clubhouse.io/augur/story/8228)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: src/modules/reporting/containers/reporting-dispute-markets.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const mapStateToProps = (state, { history }) => {
     navigateToAccountDepositHandler: () => history.push(makePath(ACCOUNT_DEPOSIT)),
     outcomes: disputeOutcomes,
     account: loginAccount.address,
-    isForking: state.universe.forkEndTime && state.universe.forkEndTime !== '0',
+    isForking: (state.universe.forkEndTime && state.universe.forkEndTime !== '0') || false,
```
Comment (justinbarry) [inline src/modules/reporting/containers/reporting-dispute-markets.js]: @bthaile Why was this needed?

Comment (bthaile) [reply src/modules/reporting/containers/reporting-dispute-markets.js]: IsForking is undefined if there isnt a forking market. Giving it a default value

Comment (justinbarry) [reply src/modules/reporting/containers/reporting-dispute-markets.js]: Got it.
